src/08/Style/index.css

```diff
@@ -71,3 +85,32 @@ div > span {
 .example4 {
   text-align: left;
 }
+
+.example5 {
+  text-align: center;
+}
+
+.contenedor {
+  display: flex;
+  flex-direction: row;
+}
```

Comment:
This solution works but there is another way using other property related with the other type of flex direction. try looking how to make columns in paragraphs.